spec.md

```diff
@@ -400,10 +407,45 @@ The following rules apply if `parameters` is included anywhere in the catalog:
 * Platforms SHOULD be prepared to support later versions of JSON schema.
 * The `$schema` key MUST be present in the schema declaring the version of JSON
 schema being used.
-* Schemas MUST NOT contain any external references.
+* Schemas MUST NOT contain any external references that are resolved to a URI
+  that requires additional fetching.
 * Schemas MUST NOT be larger than 64kB.
 
 
+###### $ref Usage
+
+It is likely that a Broker will have redundant definitions of `parameters` in
+the catalog. To simplfy this, the broker MAY provide a `root_schemas` entry. The
+Platform will need to process the catalog's `root_schemas` to fully qualify
```

Comment:
is this sentence necessary?

Reply:
Yes, because we insist that the catalog must contain the entire json schema, it breaks json normal json schema parsing tools. Our choices here will cause people to need to rewrite their parsing tools slightly.